jhub_apps/service/routes.py

```diff
@@ -166,24 +176,56 @@ async def start_server(
     """Start an already existing server."""
     logger.info("Starting server", server_name=server_name, user=user.name)
     hub_client = HubClient(username=user.name)
+    
+    hub_user = hub_client.get_user()
```

Comment:
The unit tests are failing because this function is called and its not mocked in the tests, like for example: https://github.com/nebari-dev/jhub-apps/blob/1ef9701bc6936c25afca0a10428f54a3eb9681ec/jhub_apps/tests/tests_unit/test_api.py#L86

You'd need to mock this function in the test and set the expected return value like: https://github.com/nebari-dev/jhub-apps/blob/1ef9701bc6936c25afca0a10428f54a3eb9681ec/jhub_apps/tests/tests_unit/test_api.py#L89